src/pages/users/users-details/containers/user-details-header/user-details-header.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Typography, Paper, Avatar } from '@material-ui/core';
+import { useStyles } from './user-details-header.styles';
+
+const UserDetailsHeader = ({ title, firstName, lastName, status }) => {
+  const styles = useStyles();
+
+  return (
+    <Grid container spacing={1}>
+      <Grid item xs={12}>
+        <Typography variant='h1' className={styles.detailsTitle}>
+          {title}
+        </Typography>
+      </Grid>
+      <Grid item xs={12}>
+        <Paper className={styles.userBlock}>
+          <Grid container spacing={1} className={styles.userHeader}>
+            <Grid item xs={1} className={styles.userAvatar}>
+              <Avatar>{`${firstName[0]}${lastName[0]}`}</Avatar>
```

Comment:
There it is better to use `<Avatar>{firstName[0]} {lastName[0]}</Avatar>`. In this case, string concatenation with template string is not needed